cmd/wego/main.go

```diff
@@ -48,4 +48,12 @@ func main() {
 		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
 		os.Exit(1)
 	}
+
+	//Run on all commands that are not version
+	if os.Args[1] != "version" {
```

Comment:
I wonder if we can simplify this by removing this `if` and remove the call to CheckVersion in the `version` command 🤔.  So the check can be done in only one place.

Reply:
I see what you mean. Yeah that is a very valid point.